### PR TITLE
belinda-nextjs-181-admin_page_creation

### LINKED ISSUE #181: Admin Page: Admin Page Creation
relates to: #128

### PR DESCRIPTION
**Pull request by kaimanasse15**
Resolves #181
Description:
- Added an admin page.

### CODE REVIEW

**Comment by tinpham5614** [comment]
- When you create a new pull request, you can replace the template description with your own and provide more details on your approach, implementation, and outcomes. Below is an example of a pull request description:
image

- When you mentioned, "Admin Page the mongodb has problem". Is there something that doesn't work on your side?
Let me know if you want me to walk through the environment installation.

**Comment by kaimanasse15** [comment]
Yes, I worked with Taylor yesterday. Only me that I could create the account but the data doesn’t go to my mongoldb. It is fine on the other mongodb.

**Comment by kaimanasse15** [comment]
Screenshot 2024-01-25 175545
I am Unable to login

**Review by tinpham5614 — APPROVED** [review]
Looks good for a placeholder admin page.

File: app/admin/page.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//   return (
+//     <div className="flex flex-col items-center justify-center min-h-screen">
+//       <h1>Welcome, {user?.name}!</h1>
+//       <p>Email: {user?.email}</p>
+//       <p>Profile information goes here...</p>
+//     </div>
+//   );
+// };
+
+// export default Profile;
```
Comment (bcko) [inline app/admin/page.tsx:45]: We usually don't want to have this large blocks of commented out code pushed to our main branch unless we have a good reason. Can you explain why you are leaving this comments? 
Please consider removing this from this pull request

Comment (tinpham5614) [reply app/admin/page.tsx:45]: I will go ahead and fix this.

**Review by taylorpapke — APPROVED** [review]
Looks good. I am seeing the placeholder admin page:
Screenshot 2024-01-30 at 1 02 25 AM

**Comment by kaimanasse15** [comment]
> Looks good. I am seeing the placeholder admin page: <img alt="Screenshot 2024-01-30 at 1 02 25 AM" width="618" src="https://private-user-images.githubusercontent.com/94101953/300741773-f1d9c563-8f81-40d2-ab39-762375f7387c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MDU3NTQsIm5iZiI6MTcwNjYwNTQ1NCwicGF0aCI6Ii85NDEwMTk1My8zMDA3NDE3NzMtZjFkOWM1NjMtOGY4MS00MGQyLWFiMzktNzYyMzc1ZjczODdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTMwVDA5MDQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhMDljMDkxNTdhYTVkODdjODBjYTI2YTM2Y2QzZTE4Y2NjNDg5YThiNjQxMWJjYTUwMTljNWY3MjNlNTM2YWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7FYzQFI-IchGKx2XZt81dYAa0IzzdlyBP

**Comment by kaimanasse15** [comment]
I accidentally clicked close. I will working on the admin page. I just have busy these days. Thanks Taylor and Tin.

**Pull request merged**